romancal/resample/tests/test_resample.py

```diff
@@ -1014,7 +1014,7 @@ def test_populate_mosaic_basic_different_observations(
         rotation=0,
         shape=None,
         crpix=(0, 0),
-        crval=(0, 0),
+        crval=(10, 0),
```

Comment:
Tests are still failing with `crpix=(0,0)` + `crval=(10,0)`. It does work when removing `crpix` alone (regardless of `crval`'s value).